client/my-sites/scan/threat-fixed-review-prompt/index.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@wordpress/components';
+import Gridicon from 'calypso/components/gridicon';
+import { preventWidows } from 'calypso/lib/formatting';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import QueryPreferences from 'calypso/components/data/query-preferences';
+import { dismiss } from 'calypso/state/jetpack-review-prompt/actions';
+import {
+	getIsValid,
+	getIsDismissed,
+	getExistingPreference,
+} from 'calypso/state/jetpack-review-prompt/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ThreatFixedReviewPrompt: React.FC = () => {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+	const dispatch = useDispatch();
+
+	const isReviewPromptValid = useSelector( ( state ) => getIsValid( state, 'scan' ) );
+	const isReviewPromptDismissed = useSelector( ( state ) => getIsDismissed( state, 'scan' ) );
+	const lastThreatFixedDate = useSelector(
```

Comment:
whoops! we could definitely at a `getValidFrom` selector to existing selector files, but this works great 👍